cmd/references/crtref.go

```diff
@@ -50,7 +50,7 @@ func init() {
 		"", "Description for the reference")
 	CreateCmd.Flags().StringVarP(&resourceType, "restype", "s",
 		"", "Resource type must be KeyStore or TrustStore")
-	CreateCmd.Flags().StringVarP(&refers, "refers", "r",
+	CreateCmd.Flags().StringVarP(&refers, "refers", "",
```

Comment:
why was this removed? Is it because of the global `r` flag? If yes, then removing the existing `r` for `refers` is breaking change right?